app/controllers/game.js

```diff
@@ -0,0 +1,131 @@
+const mongoose = require('mongoose');
+// var Game = require('../models/game');
+var Game = mongoose.model('Game');
+
+/**
+ * Saves game log when game session ends
+ * @returns {object} description
+ * @export { function }
+ * @param {object} req
+ * @param {object} res
+ */
+exports.createGameData = (req, res) => {
+  // save game if user is authenticated
+  if (req.decoded && req.params.id) {
+    const game = new Game(req.body);
+    game.userID = req.decoded.currUser.id;
+    game.gameID = req.params.id;
+    game.save((error) => {
+      if (error) {
+        return res.status(400).send({
+          message: 'Error! Game not saved'
+        });
+      }
+      return res.status(201).send({
+        message: 'Game Saved Successfully!'
+      });
+    });
+  }
+};
+
+/**
+ * Sorted winners function
+ * @returns {Object} array showing the winners and the frequency of wins
+ * @param {Object} gameWinners
+ */
+const sortWinners = (gameWinners) => {
+  const sortedWinners = {};
+  gameWinners.forEach((index) => {
+    sortedWinners[index] = (sortedWinners[index] || 0) + 1;
+  });
+  return sortedWinners;
+};
+
+/**
+ * getGameWinners function
+ * @returns {Object} array of the individual game winners
+ * @param {Object} gameResults
```

Comment:
`@param` should always be before `@return`. The JSDoc should follow this example
```js
/**
  * Summary/description of function
  * @param {type} parameter.
  * @return {type} return description.
  */